nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-processors/src/main/java/org/apache/nifi/processors/elasticsearch/PutElasticsearchHttpRecord.java

```diff
@@ -284,6 +306,17 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
                 return;
         }
 
+        final NullSuppression suppression;
```

Comment:
Not sure we need this variable. We can directly use ````this.nullSuppression````in the below block, no?

Reply:
Agreed that this.nullSuppression could be directly set without the need of this variable.

Reply:
Done.